client/asset/eth/eth.go

```diff
@@ -5202,10 +5202,21 @@ func (w *ETHWallet) getReceivingTransaction(ctx context.Context, txHash common.H
 		return nil, asset.CoinNotFoundError
 	}
 
-	wt := w.extendedTx(tx, asset.Receive, dexeth.WeiToGweiCeil(tx.Value()))
```

Comment:
I stopped using `extendedTx` here as this will add it to the db. I think it will be confusing for users to see some receiving transactions displayed, but not others.

Reply:
Wouldn't it have been easier to break out the conversion part from `extendedTx` to a new function